datasets/ru/egrul/crawler.py

```diff
@@ -446,6 +447,52 @@ def parse_address(context: Context, entity: Entity, el: Element) -> None:
     entity.add("address", address)
 
 
+def compile_abbreviations(context) -> List[Tuple[str, re.Pattern, List[str]]]:
+    """
+    Load abbreviations and compile regex patterns from the YAML config.
+    Returns:
+    List[Tuple[str, re.Pattern, List[str]]]: A list of tuples containing canonical abbreviations
+    and their compiled regex patterns for substitution.
+    """
+    yaml = context.dataset.config.get("types")
+    abbreviations = []
+    for canonical, phrases in yaml.items():
```

Comment:
I think inside the yaml, we wanna be more specific - what kind of types is it?
Inside the crawler, we can't call it yaml, because it isn't yaml anymore. It's now a Python Dictionary. But instead of calling it `dictionary` or `data`, let's call it what it is - types. Since this function is only about one type of type, we can be brief and call it `type`. In contrast, there are all kinda of "types" in different yaml files - that's why we want to be more specific there.

```suggestion
    types = context.dataset.config.get("company_types")
    abbreviations = []
    for canonical, phrases in yaml.items():
```